backup-parameter-store/src/main/scala/com/gu/aws/wrappers.scala

```diff
@@ -0,0 +1,81 @@
+package com.gu.aws
+
+import java.io.ByteArrayInputStream
+import com.amazonaws.auth.{ AWSCredentialsProviderChain, InstanceProfileCredentialsProvider }
+import com.amazonaws.auth.profile.ProfileCredentialsProvider
+import com.amazonaws.services.s3.AmazonS3ClientBuilder
+import com.amazonaws.services.s3.model.{ ObjectMetadata, PutObjectRequest, PutObjectResult, SSEAwsKeyManagementParams }
+import com.amazonaws.services.simplesystemsmanagement.{ AWSSimpleSystemsManagement, AWSSimpleSystemsManagementClientBuilder }
+import com.amazonaws.services.simplesystemsmanagement.model.GetParametersByPathRequest
+
+import scala.collection.JavaConverters._
+import scala.annotation.tailrec
+
+object AwsConfig {
+  val provider = new AWSCredentialsProviderChain(
+    new ProfileCredentialsProvider("frontend"),
+    InstanceProfileCredentialsProvider.getInstance())
+  val region = "eu-west-1"
+  val kmsKeyAlias: String = "arn:aws:kms:eu-west-1:642631414762:alias/FrontendConfigKey"
+}
+
+class ParameterStore {
+
+  private lazy val client: AWSSimpleSystemsManagement = AWSSimpleSystemsManagementClientBuilder
+    .standard()
+    .withRegion(AwsConfig.region)
+    .build()
+
+  def unwrapQuotedString(input: String): String = {
+    val quotedString = "\"(.*)\"".r
```

Comment:
Can't remember the rules here, but do we need to anchor pattern to end of string?

Reply:
Nope :)